float32 t1
float32 t2
float32 t3
float32 t
string type
---
string status
